mobile/src/modules/auth/store/actions.ts

```diff
@@ -192,9 +197,11 @@ export const addAccount = createActionFn<Account, Promise<Account>>(
   }
 );
 
-export const removeAccount = createActionFn<Account, Promise<void>>(
-  async (dispatch, getState, account) => {
-    dispatch(actions.removeAccount(account));
+export const removeAccount = createActionFn<RemoveAccountPayload, Promise<void>>(
+  async (dispatch, getState, removeAccountPayload) => {
+    // tslint:disable-next-line: max-line-length
+    fetch(`https://burstalerts.com/api/v1/unsubscribe/${removeAccountPayload.deviceId}/${removeAccountPayload.account.accountRS}`);
```

Comment:
What happens if this `fetch` fails? Doesn't matter?

Reply:
yeah all good to fail silently